packages/network-controller/tests/NetworkController.test.ts

```diff
@@ -139,6 +140,13 @@ describe('NetworkController', () => {
       uuidCounter += 1;
       return uuid;
     });
+
+    jest
+      .spyOn(
+        LastUpdatedAtNetworkConfigurationModule,
+        'lastUpdatedNetworkConfiguration',
+      )
+      .mockImplementation((n) => n);
```

Comment:
I need some thoughts on this.

The addition of the `lastUpdatedAt` field breaks many tests, especially when adding or updating a network. This is fine, however it also breaks tests that are marked as "if nothing is being changes > makes no updates to state".

We may need to rethink where we set the `lastUpdatedAt` field.